package.json

```diff
@@ -37,12 +36,13 @@
     "react-datepicker": "4.25.0",
     "react-dom": "16.14.0",
     "react-router-dom": "5.2.0",
-    "reactstrap": "8.10.1"
+    "react-tag-input": "6.5.4",
+    "reactstrap": "8.10.1",
+    "webextension-polyfill": "0.12.0"
```

Comment:
This dependency was removed during manifest v3 development. Should not be re-added.